pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -423,6 +424,7 @@ func buildKubeApiserverConfig(
 	genericConfig.Authorizer = kubeAuthorizer         // this is used to fulfill the kube SAR endpoints
 	genericConfig.AdmissionControl = admissionControl
 	genericConfig.RequestContextMapper = requestContextMapper
+	genericConfig.DisabledPostStartHooks.Insert(rbacrest.PostStartHookName)
```

Comment:
for RBAC, can run the test with a user in the `system:masters` group, which the RBAC registry will allow to escalate